tests/app/main/test_services.py

```diff
@@ -26,9 +24,7 @@ def supplier_service_editing_fw_params(request):
     return request.param
 
 
-# and these fixtures really are just a workaround to not being able to use plain function parametrization in unittest-
-# derived classes and should be replaced with that as soon as we're able to do so. hence the hence the oddly specific
-# long names
+# find a better way of doing ofganizing this than using oddly specific long names
```

Comment:
I'm not mad on this comment, but if we have it in we could at least fix `ofganizing`